src/app/modules/main/module.nim

```diff
@@ -338,7 +338,7 @@ method load*[T](
   enabled = true)
   self.view.model().addItem(profileSettingsSectionItem)
   if(activeSectionId == profileSettingsSectionItem.id):
-    activeSection = profileSettingsSectionItem
+    activeSection = self.view.model().getItemBySectionType(SectionType.Chat)
```

Comment:
Hm, I can't get it. If `activeSectionId` is profile why we set activeSection to `SectionType.Chat`?

Reply:
@endulab I fully agree with @MishkaRogachev, please revert this line. 

You should fix this issue [here](https://github.com/status-im/status-desktop/blob/8190eebfd490dd8e9beea6681a65dab32cb5768f/src/app/modules/main/controller.nim#L243) using a condition like `if sectionType != SectionType. ProfileSettings` which will allow saving last opened section only in case it's not a profile section.

Reply:
@saledjenic the place you are pointing to is not good. setActiveSection() function is triggered anytime the user changes module (the info about current module is saved during runtime, not when you close the app).
The place of my change is during application startup when modules are loaded and in the end of the function there is:
`# Set active section on app start
self.setActiveSection(activeSection)` 
The first active section is set here.
However I improved this change and made it more obvious:
`# Set active section on app start
  # If section is profile then open chat by default
  if activeSection.sectionType != SectionType.ProfileSettings:
    self.setActiveSection(activeSection)`
This solution works correctly, even for users w/o any chats.

Reply:
> @saledjenic the place you are pointing to is not good. setActiveSection() function is triggered anytime the user changes module (the info about current module is saved during runtime, not when you close the app).
The place of my change is during application startup when modules are loaded and in the end of the function there is

@endulab sorry to say, but seems you misunderstood my comment, please check out [this](https://github.com/status-im/status-desktop/pull/5903#pullrequestreview-989101418).